Patch so that mdstress-library finds and uses flexiblas as LAPACK/BLAS library.

@Date:   2024-07-31 
@Author: oliver.stueker@ace-net.ca

diff -r -u gromacs-ls-2016.3.orig/src/contrib/mdstress-library/CMakeLists.txt gromacs-ls-2016.3/src/contrib/mdstress-library/CMakeLists.txt
--- gromacs-ls-2016.3.orig/src/contrib/mdstress-library/CMakeLists.txt	2019-12-28 11:13:57.000000000 -0330
+++ gromacs-ls-2016.3/src/contrib/mdstress-library/CMakeLists.txt	2024-07-31 15:45:03.286224166 -0230
@@ -40,7 +40,7 @@
 INCLUDE_DIRECTORIES(include/voro++)
 
 # attempt to find LAPACK library
-FIND_LIBRARY(LAPACK_LIBRARY_SYS NAMES lapack liblapack HINTS /usr/lib REQUIRED)
+FIND_LIBRARY(LAPACK_LIBRARY_SYS NAMES lapack liblapack flexiblas libflexiblas HINTS /usr/lib REQUIRED)
 if (NOT LAPACK_LIBRARY_SYS)
     UNSET(LAPACK_LIBRARY_SYS-NOTFOUND)
     UNSET(LAPACK_LIBRARY_SYS)
@@ -53,7 +53,7 @@
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/contrib)
     ADD_SUBDIRECTORY(contrib/lapack-3.8.0)
 else()
-    FIND_LIBRARY(LAPACK_LIBRARY NAMES lapack liblapack HINTS /usr/lib REQUIRED)
+    FIND_LIBRARY(LAPACK_LIBRARY NAMES lapack liblapack flexiblas libflexiblas HINTS /usr/lib REQUIRED)
 endif()
 
 ADD_LIBRARY(mdstress SHARED ${MDSTRESS_SOURCES})
 
